### ISSUE TITLE
Downgrade version of addon

### ISSUE THREAD

**Issue by mryanlam** [ajour/ajour#558]
**Is your feature request related to a problem? Please describe.**
Occasionally an addon update breaks something and it is desirable to install an older version of that one specific addon.

**Describe the solution you'd like**
An option to specify which specific release to install, not just channel

**Describe alternatives you've considered**

**Additional context**

**Comment by tarkah** [comment]
@mryanlam We've added this to our backlog, but we haven't had much of a chance to discuss it or prioritize it yet.

It would be fairly easy to implement a "rollback" feature, where we always backup the addon before updating it to the new version. Then the user can always rollback to that previous version.

It'd be much more difficult to add the ability to revert to ANY previous version, and I'd need to look into the APIs for each repository to see if that's information we can even get. I know for a fact it's not possible on Tukui outside the two main Tukui / Elvui addons.

Which idea were you thinking as part of this?

**Comment by mryanlam** [comment]
A rollback would be sufficient for the case where an update breaks some functionality.

**Comment by tarkah** [comment]
Agreed. This will probably be the path we take once we prioritize this, since we can implement it without regard to the various repositories and their capabilities.

**Comment by casperstorm** [comment]
I'll close this issue for now, because the API does not have sufficient information to be able to do this for all addons.
Few addons has the information, but the majority won't so it will be confusing why some can, and some can't and because of this I think its best not to focus on this feature right now.
